cluster/juju/layers/kubernetes-e2e/README.md

```diff
@@ -21,7 +21,8 @@ and then relate the `kubernetes-e2e` charm.
 ```shell
 juju deploy kubernetes-core
 juju deploy cs:~containers/kubernetes-e2e
-juju add-relation kubernetes-e2e kubernetes-master
+juju add-relation kubernetes-my-worker:kube-api-endpoint kubernetes-master:kube-api-endpoint
+juju add-relation kubernetes-my-worker:kube-control kubernetes-master:kube-control
```

Comment:
`kubernetes-my-worker` -> `kubernetes-worker`? Pretty sure the worker in this example is coming from `juju deploy kubernetes-core`, hence it would get the default name.